src/screens/CreatePost/index.js

```diff
@@ -936,7 +935,7 @@ const styles = StyleSheet.create({
     height
   }),
   reminderContainer: {
-    backgroundColor: '#2F80ED',
+    backgroundColor: colors.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color value '#2F80ED' has been replaced with `colors.blue`. This is a good practice as it improves maintainability. If the color needs to be changed in the future, you only need to change it in one place.
```diff
- backgroundColor: '#2F80ED',
+ backgroundColor: colors.blue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->